### PR TITLE
Fix Style/SpaceAroundOperators

### PR DESCRIPTION
**Pull request by esasse**

### CODE REVIEW

File: lib/bundler/source/git.rb [hunk 1]
```diff
@@ -50,12 +50,7 @@ def hash
       end
 
       def eql?(o)
-        o.is_a?(Git)         &&
-        uri == o.uri         &&
-        ref == o.ref         &&
-        branch == o.branch   &&
-        name == o.name       &&
-        version == o.version &&
+        o.is_a?(Git) && uri == o.uri && ref == o.ref && branch == o.branch && name == o.name && version == o.version &&
```
Comment (segiddins) [inline lib/bundler/source/git.rb]: don't love this

**Comment by segiddins** [comment]
In general, I prefer this, but I'll let the rest of the team chime in on this one

File: lib/bundler/source/git.rb (hunk 1 above)
Comment (esasse) [inline lib/bundler/source/git.rb]: Me neither, but unless you do it, rubocop will fail. I can investigate if it's possible to rubocop ignore just this part of code, but maybe it's not a good idea.

Comment (segiddins) [inline lib/bundler/source/git.rb]: I mean we can still have them each on their own line

Comment (esasse) [inline lib/bundler/source/git.rb]: Done! :+1:

**Comment by indirect** [comment]
I wish there was a way to enforce this without breaking the places where there's multi-line alignment, but I'd rather have enforced spaces than the ability to align things across lines.

**Pull request closed without merging**